client/state/action-types.js

```diff
@@ -105,5 +109,22 @@ export const SUPPORT_USER_ACTIVATE = 'SUPPORT_USER_ACTIVATE';
 export const SUPPORT_USER_ERROR = 'SUPPORT_USER_ERROR';
 export const SUPPORT_USER_TOGGLE_DIALOG = 'SUPPORT_USER_TOGGLE_DIALOG';
 export const SUPPORT_USER_TOKEN_FETCH = 'SUPPORT_USER_TOKEN_FETCH';
+export const THEME_ACTIVATE = 'THEME_ACTIVATE';
+export const THEME_ACTIVATED = 'THEME_ACTIVATED';
+export const THEME_ACTIVATED_WITH_CUSTOMIZER = 'THEME_ACTIVATED_WITH_CUSTOMIZER';
+export const THEME_CLEAR_ACTIVATED = 'THEME_CLEAR_ACTIVATED';
+export const THEME_CUSTOMIZE = 'THEME_CUSTOMIZE';
+export const THEME_DETAILS_RECEIVE = 'THEME_DETAILS_RECEIVE';
+export const THEME_DETAILS = 'THEME_DETAILS';
```

Comment:
`THEME_DETAILS_SHOW` vs `THEME_CUSTOMIZE` et al -- technically, all just navigate to a different page.